cves/CVE-2017-11665.yml

```diff
@@ -130,7 +125,11 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The time between the VCC and the most recent fix is 8 long years
+    from July 31st, 2009 to July 28th, 2017
+    this file looks like it's had many incremental changes over the years of 
```

Comment:
I need more specifics than this - spend some more time making observations about the development during that time.